app/views/claims/schools/mentors/index.html.erb

```diff
@@ -18,7 +18,7 @@
           <% table.with_body do |body| %>
             <% @mentors.each do |mentor| %>
               <% body.with_row do |row| %>
-                <% row.with_cell(text: govuk_link_to(mentor.full_name, "#")) %>
+                <% row.with_cell(text: govuk_link_to(mentor.full_name, claims_school_mentor_path(id: mentor.id))) %>
```

Comment:
We don't need to explicitly call `id` here. Rails handles Active Record instances into params for us.

```suggestion
                <% row.with_cell(text: govuk_link_to(mentor.full_name, claims_school_mentor_path(mentor))) %>
```

Side note, Rails calls `#to_param` behind the scenes. Therefore, if you were to add slugged urls, passing `_path(record)` would require no changes, whereas `_path(record.id)` would always pass through the `id` to the url builder and not the custom _param_ value.